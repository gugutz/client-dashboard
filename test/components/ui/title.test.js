import React from 'react'
import renderer from 'react-test-renderer'
import Title from '../../../src/components/ui/title'

describe('<Title /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Title>
Some cool title
                                 </Title>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
