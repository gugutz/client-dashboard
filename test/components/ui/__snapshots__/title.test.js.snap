// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> component should render correctly 1`] = `
<h1
  className="title"
>
  Some cool title
</h1>
`;
